numbers.csv

```diff
@@ -4,3 +4,4 @@ Sue,42
 Diana,27
 Jason,9
 Sophia, 10
+Daniel, 2
```

Comment:
Add missing newline at end of file.  Some unix programs can only read a line if it ends with a newline, and if the last line has no trailing newline, they'll ignore that line.